app/models/event.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  scope :past, -> { where('date <= ?', Date.today) }
+  scope :upcoming, -> { where('date >= ?', Date.today) }
+
```

Comment:
- [ ] **[OPTIONAL]** For better user experience, add date validation to prevent the user from creating events in the past.